tests/test_saveload.py

```diff
@@ -185,7 +185,8 @@ def test_saveload_modelresult_exception():
     clear_savefile(SAVE_MODEL)
 
 
-def test_saveload_modelresult_roundtrip(method='leastsq'):
+@pytest.mark.parametrize("method", ['leastsq', 'nelder', 'powell', 'cobyla', 'bfgsb', 'differential_evolution', 'brute', 'basinhopping', 'ampgo', 'shgo', 'dual_annealing'])
```

Comment:
remember to stay within the line-length-limit if possible, this should likely become:
```
@pytest.mark.parametrize("method", ['leastsq', 'nelder', 'powell', 'cobyla',
                                    'bfgsb', 'differential_evolution', 'brute',
                                    'basinhopping', 'ampgo', 'shgo',
                                    'dual_annealing'])
```